acceptance/tests/options/no_color.rb

```diff
@@ -0,0 +1,16 @@
+# This test is intended to ensure with --debug and --no-color, facter does not send any escape sequences
+# to colorize the output
+test_name "C99975: --debug and --no-color command-line options should print DEBUG messages without color escape sequences" do
+  tag 'risk:medium'
```

Comment:
This risk seems appropriate as is because the user impact is higher if the removal of escape sequences is not honored (e.g. script-based parsing of output).